web/src/utils/schema/editor.ts

```diff
@@ -63,6 +65,16 @@ class EditorInterface {
     ));
 
     this.transactionTracker = new MeditorTransactionTracker(this);
+
+    // save model to local storage on changes
+    watch(() => this.getModel(), (newModel: DandiModel) => {
+      setModelLocalStorage(newModel.id as string, newModel);
+    });
```

Comment:
Should this be debounced, so we're not writing to local storage every time the user hits a key? I'd think a debounce period of even just a second would be sufficient.

Reply:
The model is updated (and thus gets written to local storage) not on every key press, but on each `input` event from vjsf, i.e. when a form is unfocused.